.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules/
 .DS_Store
 .build/
 .coveralls.yml
-*.iml
\ No newline at end of file
+*.iml
+package-lock.json
```

Comment:
to nie jest dobra praktyka. locki powinny być wcomitowane

Reply:
btw. będziemy mieli package lock i yarn lock? :)

Reply:
tak jak Ci pisałem, package-locka z npm nie mamy, i go ignorujemy, używajmy yarna :)
A kto używa npm ten trąba :p, chociaż imo npm audit to fajny bajerek